### PR TITLE
West Midlands Class 3 - Maria Chira - Git & GitHub - Week 1

### PR DESCRIPTION
**Pull request by Maria-Chira**
I've replaced the wrong answers with the correct ones.

My details:
- Your Name: Maria Chira
- Your City: West Midlands
- Your Slack Name: Maria Georgiana Chira 

# Homework Details

- Module: Git & GitHub
- Week: 1

-----
[View rendered .github/pull_request_template.md](https://github.com/Maria-Chira/GitHomeworkFixErrors/blob/main/.github/pull_request_template.md)
[View rendered homework.md](https://github.com/Maria-Chira/GitHomeworkFixErrors/blob/main/homework.md)

### CODE REVIEW

File: homework.md [hunk 1]
```diff
@@ -4,38 +4,38 @@
...
 
-Hyper Tall Massive Language
+HTML stand for Hypertext Markup Language.
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+CSS helps us to change the style of web pages.
```
Comment (dotdashdon) [inline homework.md:32]: You could expand this answer to include layout, organisation and colour of a website.

**Comment by dotdashdon** [comment]
Hard to find any fault with this Maria! I think you've answered really well and in full too.

**Comment by Krrish-CYF** [comment]
Hi Maria, I've reviewed your assessment, and i like how you keep it simple and focused with lots of details.You have done fantastic work here like always and everything's seems to be prefect .

**Review by garvey-marcus-code — COMMENTED** [review]
Review completed, good detailed answers have been given.

**Pull request closed without merging**